pkg/api/converterfromapi.go

```diff
@@ -111,6 +154,16 @@ func convertPropertiesToVLabs(api *Properties, vlabsProps *vlabs.Properties) {
 	convertCertificateProfileToVLabs(&api.CertificateProfile, &vlabsProps.CertificateProfile)
 }
 
+func convertLinuxProfileToV20160930(api *LinuxProfile, v20160930 *v20160930.LinuxProfile) {
+	v20160930.AdminUsername = api.AdminUsername
+	v20160930.SSH.PublicKeys = []struct {
+		KeyData string `json:"keyData"`
+	}{}
+	for _, d := range api.SSH.PublicKeys {
+		v20160930.SSH.PublicKeys = append(v20160930.SSH.PublicKeys, d)
+	}
```

Comment:
This is different from what's there in: convertLinuxProfileToV20160330.
Do these need to be different and why?